packages/@uppy/xhr-upload/src/index.js

```diff
@@ -167,7 +167,7 @@ module.exports = class XHRUpload extends BasePlugin {
   }
 
   // eslint-disable-next-line class-methods-use-this
-  addMetadata (formData, meta, opts) {
+  #addMetadata (formData, meta, opts) {
     const metaFields = Array.isArray(opts.metaFields)
       ? opts.metaFields
       : Object.keys(meta) // Send along all fields by default.
```

Comment:
Declare this function outside of the class body instead, if it's not using `this` it doesn't need to be in the class body if we don't want to expose it.